### PR TITLE
Merge the Signed Data gateway in the http-gateways.md doc

### PR DESCRIPTION
**Pull request by wkande**
@amarthadan - So this PR still needs attention to the response section of the `http-gateways.md` doc. The Docker deliver image for dev from the CI was not working. I message Aaron to see if he can help.

I made an assumption that a CURL call for both the HTTP Gateway and the HTTP Signed Data Gateway are the same structure. 

Please preview the preview build show in the comments.

### CODE REVIEW

File: docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+also use
+[Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install)
+(WSL2) to run CURL for Linux.
+
+| parameter         | in     | CURL options                                             |
+| ----------------- | ------ | -------------------------------------------------------- |
+| Content-Type      | header | `-H 'Content-Type: application/json'`                    |
+| x-api-key         | header | `-H 'x-api-key: 8d890a46-799d-48b3-a337-8531e23dfe8e'`   |
+| endpointId        | path   | `0xf466b8feec...99e9f9f90453c`                           |
+| &lt;user-defined> | body   | `-d '{"parameters": {"param1": "string", "param2": 5}}'` |
```
Comment (amarthadan) [inline docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md]: As I wrote you on Slack, there's a difference between the two when it comes to the `parameters` parameter.
HTTP Gateway's is called `parameters` and provides values for the parameters in a JSON object format (like above)
HTTP Signed Data Gateway's is called `encodedParameters` and provides parameters' description in the encoded form, somethkig like `{"encodedParameters": "0x3173737300....000"}`.

File: docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+- `encodedValue` - the encoded bytes value that is sent as payload in the
+  response transaction on chain
+
+:::
+
+::: tab HTTP Signed Data Gateway
+
+:::
+
+::::
```
Comment (amarthadan) [inline docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md]: You can see an example response from the HTTP Signed Data Gateway [here](https://github.com/api3dao/airnode/pull/870#pullrequestreview-893537683). That will most likely change but for now, this is how it looks like.

File: docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+```
+
+## Gateway URLs
+
+A gateway URL is generated for each gateway (when enabled) when Airnode is
+deployed. You can obtain the URLs `api.httpGatewayUrl` and
+`api.httpSignedDataGatewayUrl` from the
+[receipt.json](../../../reference/deployment-files/receipt-json.md) file
+returned by the Airnode deployer. They are also available as part of the payload
+sent from Airnode's [heartbeat](./heartbeat.md) to your specified heartbeat URL.
```
Comment (amarthadan) [inline docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md:113]: This is actually a good point. We haven't added the new gateway URL into the heartbeat :sweat_smile: I'm going to do that now and I'll also update the heartbeat docs.

**Review by amarthadan — CHANGES_REQUESTED** [review]
This looks quite nice to me :+1:
@bbenligiray Can you take a look as well?

File: docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md (hunk 1 above)
Comment (bbenligiray) [reply docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md]: In my mind `parameters` always means encoded parameters and you should specify `decodedParameters` when using them.

File: docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md (hunk 3 above)
Comment (wkande) [reply docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md:113]: I will leave the note about the heartbeat then, thanks

File: docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md (hunk 1 above)
Comment (wkande) [reply docs/airnode/v0.5/grp-providers/guides/build-an-airnode/http-gateways.md]: It has been changed to `encodedParameters`.

**Comment by wkande** [comment]
I will change `encodedParameters` back to `parameters` when and if the code changes.

**Pull request merged**